clean_tmp.sh

```diff
@@ -0,0 +1 @@
+find /tmp/* -prune -type d -mtime +0.084 -exec rm -r {} \;
```

Comment:
@haroldrubio do you know is there a particular pattern to directory names which get created? I'm worried removing everything from `/tmp` might cause problems elsewhere

Reply:
Yup the directories start with `tmp`, so `/tmp/tmp71atajqf`. Is would `/tmp/tmp*` be the correct regex?

Reply:
I believe so.